en/lessons/basics/control-structures.md

```diff
@@ -1,207 +1,246 @@
 
-Another neat feature of `case/2` is its support for guard clauses:
+### Syntax
 
-_This example comes directly from the official Elixir [Getting Started](https://elixir-lang.org/getting-started/case-cond-and-if.html#case) guide._
+```
+result = case (expression) of
+    (pattern1) -> (expression1) 
+    (pattern2) -> (expression2)
+    ...
+    _          -> (expressionLast)
+```
 
-```elixir
-iex> case {1, 2, 3} do
-...>   {1, x, 3} when x > 0 ->
-...>     "Will match"
-...>   _ ->
-...>     "Won't match"
-...> end
-"Will match"
+- The expression between the `case` and `of` keywords is evaluated
+- Its result is looked up in the list of patterns
+- For the first pattern that matches, its expression is evaluated
+- The underscore (`_`) is a catch-all pattern that will always match
+
+### Examples
+
+```haskell
+-- Offer some helpful advice, given the weather
+weatherAdvice :: Text -> Text
+weatherAdvice weatherDescription = 
+    case weatherDescription of
+        "fair"     -> "It'll be nice today"
+        "rainy"    -> "Bring an umbrella"
+        "sunny"    -> "Wear sunscreen"
+        "freezing" -> "You'll need a coat"
+        _          -> "Not sure"
 ```
 
-Check the official docs for [Expressions allowed in guard clauses](https://hexdocs.pm/elixir/guards.html#list-of-allowed-expressions).
+### Usage
 
-## `cond`
+Case statements are used commonly for intermediate values, and to decide the path
+that code will take
 
-When we need to match conditions rather than values we can turn to `cond/1`; this is akin to `else if` or `elsif` from other languages:
+The *Pattern Matching* feature they use will be discussed more in the next
+lesson
 
-_This example comes directly from the official Elixir [Getting Started](https://elixir-lang.org/getting-started/case-cond-and-if.html#cond) guide._
+## Guards
 
-```elixir
-iex> cond do
-...>   2 + 2 == 5 ->
-...>     "This will not be true"
-...>   2 * 2 == 3 ->
-...>     "Nor this"
-...>   1 + 1 == 2 ->
-...>     "But this will"
-...> end
-"But this will"
-```
+Guards allow you to choose between one of many possible conditions; they are
+similar to an `if`-`else if` structure
 
-Like `case/2`, `cond/1` will raise an error if there is no match.
-To handle this, we can define a condition set to `true`:
+### Syntax
 
-```elixir
-iex> cond do
-...>   7 + 1 == 0 -> "Incorrect"
-...>   true -> "Catch all"
-...> end
-"Catch all"
+```
+result
+    | (boolExpression1) = (resultExpression1)
+    | (boolExpression2) = (resultExpression2)
+    | ...
+    | otherwise = (resultExpressionLast)
 ```
 
-## `with`
-
-The special form `with/1` is useful when you might use a nested `case/2` statement or situations that cannot cleanly be piped together. The `with/1` expression is composed of the keywords, the generators, and finally an expression.
+- Each expression on the right side of the `|` character evaluates to a `Bool`
+- For the first one which evaluates to `True`, its expression is evaluated to
```

Comment:
Wouldn't be nice to me more explicit: "The first pattern that matches -- and only it -- has its expression evaluated to"?